cmd/vic-machine/validate/validator.go

```diff
@@ -222,6 +273,7 @@ func (v *Validator) getResources(ctx context.Context) error {
 		v.Session.Pool, err = finder.ResourcePoolOrDefault(ctx, v.ResourcePoolPath)
 		if err != nil {
 			errs = append(errs, fmt.Sprintf("Failure finding pool (%s): %s", v.ResourcePoolPath, err.Error()))
+			v.suggestComputeResource()
```

Comment:
Have the suggest method in this method. And verify all the scenarios you described in the PR.